plugin/evm/validators_manager_test.go

```diff
@@ -0,0 +1,65 @@
+package evm
+
+import (
+	"encoding/binary"
+	"math/rand"
+	"testing"
+
+	"github.com/flare-foundation/flare/ids"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func randArray() [20]byte {
+	data := [20]byte{}
+	rand.Read(data[:])
+
+	return data
+}
+
+func randUint64() (uint64, error) {
+	data := make([]byte, 8)
+	_, err := rand.Read(data)
+	return binary.LittleEndian.Uint64(data), err
+}
```

Comment:
Why not just use [rand.Uint64](https://pkg.go.dev/math/rand#Uint64)? 🤔

Reply:
Because I forgot, thanks!

Reply:
Here -> https://github.com/flare-foundation/coreth/pull/21/commits/b5d59c2475c2f081d8ccb493da8daedbd87067ff#diff-7cc8115837d92067c0e48165d5b49fcd63797639f8680ceee734caa4983e9b32R490
its using simply rand.Uint64 now
Thanks!